ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -1599,13 +1599,15 @@ private void CheckRangeNotInTable(XLRange range)
 
         private string GetNewTableName(string baseName)
         {
+            var existingSheetNames = this.Workbook.Worksheets.SelectMany(ws => ws.Tables).Select(t => t.Name).ToList();
```

Comment:
It's `existingTableNames`, isn't it?

Reply:
Besides, I'd suggest to make it `new HashSet<string>(
                Workbook.Worksheets.SelectMany(ws => ws.Tables).Select(t => t.Name),
                StringComparer.OrdinalIgnoreCase);`

This will make searching slightly more efficient in cases when the user dumps out _many_ tables.

Reply:
Pity `.ToHashSet()` appeared only in .NET 4.7.2.